config/initializers/content_security_policy.rb

```diff
@@ -38,8 +40,14 @@
                       :https,
                       'https://td.doubleclick.net'
 
-  # Specify URI for violation reports
+
+  policy.font_src     :self,
+                      :https,
+                      'moz-extension:',
```

Comment:
Do we really need to add fonts from extensions? Citing https://www.troyhunt.com/add-ons-extensions-and-csp-violations-playing-nice-with-content-security-policies/ as a reason why we should leave these out and ask extension makers to handle it themselves.

Reply:
makes sense. Lets remove that